oci/utils_unix.go

```diff
@@ -99,7 +100,7 @@ func getDevices(path, containerPath string) ([]specs.LinuxDevice, error) {
 	return out, nil
 }
 
-func deviceFromPath(path string) (*specs.LinuxDevice, error) {
+func DeviceFromPath(path string) (*specs.LinuxDevice, error) {
```

Comment:
Could you add godoc comment?

Reply:
Ah, yes, forgot; I updated and added a godoc